sda-svc/templates/networkpolicies.yaml

```diff
@@ -105,13 +105,20 @@ spec:
     ports:
     - protocol: TCP
       port: {{ .Values.global.doa.servicePort }}
+  egress:
+  - to:
+    - ipBlock:
+        cidr: {{ .Values.global.doa.egressips }}
+    ports:
+    - protocol: TCP
+      port: {{ .Values.global.doa.egressPort }}
```

Comment:
Is it not necessary to have `policyTypes` defined? Genuinely not sure

Reply:
seems to add them when I do helm charts install, we can explicitly define them